src/users/user.service.ts

```diff
@@ -36,10 +38,20 @@ export class UserService {
       cursor,
       where,
       orderBy,
+      select: {
+        id: true,
+        name: true,
+        login: true,
+        password: false,
```

Comment:
O default já é `false`. Mas foi "Just to be extra sure"? kkkkkkkkkkkkkkk

Reply:
Na hora que eu fiz esse ai eu não manjava que o default era false. Nos novos selects não to colocando essa BURRADA 😭